File: app/src/main/java/ru/prsolution/winstrike/mvp/models/ConfirmModel.java
package ru.prsolution.winstrike.mvp.models;

/**
 * Created by designer on 07/03/2018.
 */

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ConfirmModel {

    @SerializedName("username")
    @Expose
    private String phone;

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }
}

File: app/src/main/kotlin/ru/prsolution/winstrike/domain/models/ConfirmModel.kt
package ru.prsolution.winstrike.domain.models

/**
 * Created by oleg on 07/03/2018.
 */

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ConfirmModel {

	@SerializedName("username")
	@Expose
	var phone: String? = null
}


